es6/tests/base.js

```diff
@@ -924,3 +926,93 @@ describe("Serialization", function() {
 		JSON.stringify(doc);
 	});
 });
+
+describe("Docxtemplater v4 tests", function() {
+	it("should work when modules are attached with new constructor", function() {
```

Comment:
Can you remove the " with new constructor" in the if blocks ?

Reply:
Done